source/documentors/references/quick_reference_rst.rst

```diff
@@ -172,6 +221,111 @@ Use the following code:
 
 .. seealso:: To see alternative ways of defining tables, visit the `RST documentation about this topic <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#tables>`_.
 
+Add a Substitution
+******************
+
+In RST, a *substitution* serves as a variable which you can set a value for once, then use repeatedly. This is useful for words or phrases that are used often, as it enables you to edit the value once and change it everywhere.
+
+You also need to use substitutions for inline images, as explained below.
+
+Substitutions are all kept in the source/substitutions.txt file in the documentation project on GitHub.
+
+Copy the format for the substitution as needed.
+
+.. code-block:: RST
+
+  .. |variable name| replace:: value
+
+  .. |Platform name| replace:: Open edX
+
+You then add the *variable name* inline in the topic.
+
+.. code-block:: RST
+
+  A line of text with an |variable name| inserted.
+
+
+Add a Sidebar
+*************
+
+You can add any content in a sidebar. Open edX uses sidebars for image thumbnails, videos, and other notes.
```

Comment:
```suggestion
You can add any content in a sidebar. Open edX documentation uses sidebars for image thumbnails, videos, and other notes.
```

Use Open edX as an adjective. Also, the site already has sidebars, should we be encouraging use of more?

Reply:
It is the same case as the substitution. I added it here to keep all the information on the previous template page. Still, I agree with you that adding more sidebars to the repository can be excessive, as we already have it. However, in the case of parallel repositories like the one for Aspects, it could be helpful to have different tools and references.
I have also seen these sidebars used in glossaries. I vote to keep it.